stacks/textToSpeechUsingElevenLabs/index.test.txt

```diff
@@ -0,0 +1,5 @@
+test('Convert this text to speech: Let your curiosity guide you', async () => {
+  const response = stack("Convert this text to speech: Let your curiosity guide you");
```

Comment:
do this instead
const text = "Hello World"
const pathToAudio = "path/to/audio"
stack(
"Convert this text to speech: Let your curiosity guide you"
{
  in: {
   text,
   pathToAudio
}
)